packages/ckeditor5-find-and-replace/src/findandreplaceui.js

```diff
@@ -104,8 +104,12 @@ export default class FindAndReplaceUI extends Plugin {
 		} );
 
 		editor.keystrokes.set( 'Ctrl+F', ( data, cancelEvent ) => {
-			dropdown.isOpen = true;
-			cancelEvent();
+			const findCommand = editor.commands.get( 'find' );
```

Comment:
You needn't look for the command. `isEnabled` is bound directly to `dropdown` (in line 58 above).